src/makeInfinityAdder.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const adderFn = (value) => {
+    if (value === undefined) {
```

Comment:
you have done a double check the undefined. I think, you can do it without function adderFn